lib/matplotlib/widgets.py

```diff
@@ -1768,7 +1784,7 @@ def onmove(self, event):
             return
         if event.inaxes not in self.axes:
             return
-        if not self.canvas.widgetlock.available(self):
+        if not event.canvas.widgetlock.available(self):
```

Comment:
I *think* the point here is to ignore mouse interactions that are already handled by some other widget, not so much to prevent overdrawing (hence just checking the event canvas), but I'm not that sure and could instead check all canvases if you think that's correct?  Just let me know.

Reply:
That is also my understanding.

Further, if you have more than one thing doing blitting on a given figure you are going have a really bad time because the caches are going to be out of sync.

Reply:
Looking at the history, this has existed since 2006, but unfortunately came in through an unrelated commit (or at least one that doesn't explain why). I'll trust that this makes sense here.